ocaml/xapi/message_forwarding.ml

```diff
@@ -1334,7 +1378,8 @@ module Forward = functor(Local: Custom_actions.CUSTOM_ACTIONS) -> struct
 							end
 						else
 							(* if we're nt suspended then just forward to host that has vm running on it: *)
-							forward_vm_op ~vm in
+							do_op_on ~host:host
```

Comment:
Ah, I was imagining that we wouldn't even bother doing anything if the VM was already shutdown here, but I see there's no logic to implement that yet, so I'm happy with the do_op_on